packages/eslint-plugin/src/configs/react-config.js

```diff
@@ -28,6 +28,17 @@ module.exports = {
     '@griffel/hook-naming': 'error',
     '@griffel/no-shorthands': 'error',
     '@griffel/styles-file': 'error',
+    'no-restricted-globals': [
```

Comment:
Probably worth considering being more restrictive. I don't believe these rules prohibit using `requestAnimationFrame` for example.

[Window API](https://developer.mozilla.org/en-US/docs/Web/API/Window)

Reply:
Follow up question: how should we get a reference to `window` and `document`?

Fluent ships a [WindowProvider package](https://github.com/microsoft/fluentui/tree/master/packages/react-window-provider) that is a React context with two hooks: `useWindow()` and `useDocument()`. This package can be used with either Fluent v8 or v9 (or anything else for that matter).

Fluent v9 has a `useFluent()` hook that provides access to "[targetDocument](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-shared-contexts/src/ProviderContext/ProviderContext.ts#L8)". This method is specific to Fluent v9.

While `useFluent()` does not provide a way to get a "targetWindow", [`document.defaultView`](https://developer.mozilla.org/en-US/docs/Web/API/Document/defaultView) provides a way to access the `window` object for a `document` so perhaps we already have everything we need. This needs further investigation.